pkg/autodiscover/autodiscover.go

```diff
@@ -153,11 +153,19 @@ func DoAutoDiscover(config *configuration.TestConfiguration) DiscoveredTestData
 	data.AllInstallPlans = getAllInstallPlans(oc.OlmClient)
 	data.AllCatalogSources = getAllCatalogSources(oc.OlmClient)
 	data.Namespaces = namespacesListToStringList(config.TargetNameSpaces)
-	data.Pods, data.AllPods = findPodsByLabel(oc.K8sClient.CoreV1(), podsUnderTestLabelsObjects, data.Namespaces)
+	data.Pods, data.AllPods, err = findPodsByLabel(oc.K8sClient.CoreV1(), podsUnderTestLabelsObjects, data.Namespaces)
+	if err != nil {
+		log.Error("Cannot get resource quotas, err: %v", err)
```

Comment:
Error string seems to be a copy/paste for resource groups.  This can be something like `Cannot get pods`.